hw/bsp/gd32vf103_longan_nano/system_gd32vf103.c

```diff
@@ -0,0 +1,1256 @@
+/*!
```

Comment:
I just notice this file in the bsp, is there any reason that you don't use the `system_gd32vf103.c` included by the firmware library instead since this file is mostly "fixed" per mcu.

Reply:
There is a [define block](https://github.com/hathach/tinyusb/blob/3eb54d878af0492a85fab2811b5f84187c2f703c/hw/bsp/gd32vf103_longan_nano/system_gd32vf103.c#L46-L60) at the beginning of the file that where the system clock is selected by commenting and uncommenting the correct define. Which is a design flaw in my opinion because the define could be set outside. The current design is meant to copy the file into your project and ship it with your sources. While this is not optimal I would trim down the file quite a bit.

Reply:
I agree, having manually def/undef clock selection is a really bad design, they can just wrap them around `#ifndef` or simply parameterize it will make thing more portable any easier to upgrade. So yeah, I totally agree with your choice here.